doc/rest.md

```diff
@@ -458,7 +464,7 @@ JSON array of objects representing posted maps with following structure:
 - **url**: String. URL of the map. It points to [GET Workspace Map](#get-workspace-map).
 
 ### DELETE Workspace Maps
-Delete existing maps and all associated sources, including map-composition JSON file and map thumbnail for all mapss in the workspace. Only maps on which user has [write access right](./security.md#access-to-multi-publication-endpoints) are deleted.
+Delete existing maps and all associated sources, including map-composition JSON file and map thumbnail for all maps in the workspace. [Asynchronous tasks](async-tasks.md) of affected maps are aborted. Only maps on which user has [write access right](./security.md#access-to-multi-publication-endpoints) are deleted.
```

Comment:
'Asynchronous tasks of affected maps are aborted.' -> 'Currently running asynchronous tasks of affected maps are aborted.'